packages/debug/src/browser/debug-configuration-manager.ts

```diff
@@ -191,14 +195,23 @@ export class DebugConfigurationManager {
     }
 
     async openConfiguration(): Promise<void> {
-        const { model } = this;
+        const model = this.getModel();
         if (model) {
             await this.doOpen(model);
         }
     }
 
     async addConfiguration(): Promise<void> {
-        const { model } = this;
+        let rootUri: URI | undefined = undefined;
+        if (this.workspaceService.isMultiRootWorkspaceOpened) {
```

Comment:
This field doesn't actually guarantee that there are multiple roots, only that the the workspace is defined in a workspace file, so the `selectRootUri` should have special handling (not show the quick pick) in case of only 1 root (or 0).